client/src/App.tsx

```diff
@@ -1,25 +1,20 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './index.css';
-import { RouterProvider, createBrowserRouter } from 'react-router-dom';
 import { Login, SignUp } from './pages';
 import SplashPage from './pages/SplashPage/Splash';
-
-const router = createBrowserRouter([
-  {
-    path: '/',
-    element: <SplashPage />,
-  },
-  {
-    path: '/login',
-    element: <Login />,
-  },
-  {
-    path: '/signup',
-    element: <SignUp />,
-  },
-]);
+import Menu from './components/Menu';
 
```

Comment:
why you moved to an old version (method) to implement the router!
this is the newest method to do that (in react router dom docs..)